ehrql/measures/measures.py

```diff
@@ -300,6 +306,31 @@ def configure_dummy_data(self, *, population_size):
         """
         self.dummy_data_config.population_size = population_size
 
+    def configure_disclosure_control(self, *, enabled=True):
+        """
+        Configure disclosure control.
+
+        By default, the numerators and denominators of measures generated from real and
+        dummy tables, and by the dummy data generator, are subject to disclosure
+        control. First, values less than or equal to `7` are replaced with `0`
+        (suppressed); then, values are rounded to the nearest `5`.
```

Comment:
Do we need to wrap numbers in backticks?

Reply:
We don't. No idea where those came from.